File: 2169. Count Operations to Obtain Zero/solution.java
class Solution {
  public int countOperations(int num1, int num2) {
    int operations = 0;
    while (num1 != 0 && num2 != 0){
      if (num1 == num2){
        operations += 1;
        break;
      }
      if (num1 >= num2){
        num1 -= num2;
      }
      else if (num2 >= num1){
        num2 -= num1;
      }
      operations += 1;
    }
    
    return operations;
  }
}

File: 2169. Count Operations to Obtain Zero/solution.py
class Solution(object):
  def countOperations(self, num1, num2):
    """
    :type num1: int
    :type num2: int
    :rtype: int
    """
    operations = 0
    while num1 != 0 and num2 != 0:
      if num1 == num2:
        operations += 1
        break
      if num1 >= num2:
        num1 -= num2
      elif num2 >= num1:
        num2 -= num1
      operations += 1
    
    return operations